fdbserver/MoveKeys.actor.cpp

```diff
@@ -1498,7 +1501,18 @@ void seedShardServers(Arena& arena,
 	tr.read_snapshot = 0;
 	tr.read_conflict_ranges.push_back_deep(arena, allKeys);
 
+	bool txs_team_set = false;
 	for (const auto& [s, _] : servers) {
+		std::vector<UID> team = { s.id() };
+		Key teamListKey = storageServerListToTeamIdKey(team);
+
+		// Create teams
+		auto teamId = txs_team_set ? deterministicRandom()->randomUniqueID() : ptxn::txsTeam;
```

Comment:
The `ptxn::txsTeam` should be assigned to shard responsible for `\xff` key space. So you are assigning the first one in `servers` for `ptxn::txsTeam`, which is different from the loop in line 1569, where the last server is for `\xff` key space.